lib/discord.go

```diff
@@ -242,6 +330,44 @@ func doDiscordReq(ctx context.Context, path string, method string, body io.ReadC
 
 		RequestHistogram.With(map[string]string{"route": route, "status": status, "method": method, "clientId": identifier.(string)}).Observe(elapsed)
 	}
+
+	if wsProxy != "" {
+		if path == "/api/gateway" || path == "/api/v9/gateway" || path == "/api/gateway/bot" || path == "/api/v10/gateway/bot" {
```

Comment:
I think its safe to use something like this here:
```suggestion
		if strings.HasSuffix(path, "/gateway") || strings.HasSuffix(path, "/gateway/bot") {
```